src/AvatarButton/README.md

```diff
@@ -51,6 +51,25 @@ A button that contains the user’s Avatar.
 </Dropdown>
 ```
 
+### With Dropdown (RTL)
```

Comment:
Curious about the need to change anything from the props API to support RTL on this component? As a consumer, would I need to conditionally pass `alignLeft` when `dir="rtl"` vs. the component handling the styles automatically?

Reply:
removed